README.md

```diff
@@ -25,6 +25,12 @@ A tldr version follows:
 * Make your changes.
 * To test your changes, run `bundle exec jekyll serve` (or
   `jekyll serve -w --force_polling`)
+* Prior to submitting, run automatic link validation:
+
+  ```
+  bundle exec jekyll build
+  bundle exec htmlproofer --empty-alt-ignore _site --url-ignore "#"
```

Comment:
This is just the first step, we definitely need something more automated also.

I don't think we should fail a Travis build on this; it's a little too uncontrollable to have the build fail on the availability on some external website. But I plan on enabling daily builds that will check the links using the new Crons feature in Travis. I'm testing that setup for a few days on my fork, and will send a PR for that if it works out.